### PR TITLE
Lnd refactor

### PR DESCRIPTION
**Pull request by krtk6160**

### CODE REVIEW

**Comment by krtk6160** [comment]
Merging this with failing status due to [bug](https://github.com/helm/chart-testing/pull/300) in the chart-testing tool

File: charts/lnd/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```
Comment (nicolasburtey) [inline charts/lnd/templates/NOTES.txt:1]: I think this should be expended:

how to get the TLS, macaroon, execute a lncli on the pod, etc.

File: charts/lnd/templates/hpa.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
```
Comment (nicolasburtey) [inline charts/lnd/templates/hpa.yaml:1]: not needed?

Comment (nicolasburtey) [reply charts/lnd/templates/hpa.yaml:1]: (ie: not compatible with lnd)

File: charts/lnd/templates/lnd-pass-secret.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ printf "%s-pass" (include "lnd.fullname" .) }}
+  labels:
+    {{- include "lnd.labels" . | nindent 4 }}
+stringData:
+  password: {{ default "randAlphaNum 6" .Values.walletPassword | quote }}
```
Comment (nicolasburtey) [inline charts/lnd/templates/lnd-pass-secret.yaml:8]: there should be a lot more characters because 6 can easily be brute force. 24 at least.

**Review by nicolasburtey — COMMENTED** [review]
I have a couple of comment on the PR

**Comment by krtk6160** [comment]
> I have a couple of comment on the PR

Will address in a new PR.

**Pull request merged**